### ISSUE TITLE
Vote passed: apply for 501c3 fiscal sponsorship under OC

### ISSUE THREAD

**Issue by mrmetaverse** [omigroup/omigroup#376]
Vote passed: apply for 501c3 fiscal sponsorship under OC

![image](https://user-images.githubusercontent.com/63426722/230152508-939acb0e-81d3-4ae3-9d22-41609f6ec51e.png)

Beware: 
- there are compliance obligations (same as existing, be transparent about finances, can't spend money on lobbying etc. all normal things a nonprofit must 
- cost: we give up a % of grants to our fiscal sponsor. Its a big job to hire an accountant for our own non profit. This cost is 10% per grant that comes in. It is a lot more complex to convert to proper 501c3 (10k to sign up, plus accountant fees, etc.)

_Originally posted by @mrmetaverse in https://github.com/omigroup/omigroup/discussions/375#discussioncomment-5534936_

Deadline: TWO WEEKS :)

**Comment by mrmetaverse** [comment]
LOE: 

- Likely collaborative work, Josephine made a document. 
- Submitting is going to take whoever has access to the account.

**Comment by mrmetaverse** [comment]
![image](https://user-images.githubusercontent.com/63426722/231516443-3bfc458a-2028-46a1-9f42-b0cc6df578ac.png)


https://docs.opencollective.foundation/getting-started/how-to-apply/application-questions

https://docs.opencollective.foundation/getting-started/how-to-apply

**Comment by mrmetaverse** [comment]
while following [these instructions](https://docs.google.com/document/d/1_i1abJ0Log9UyqEjQiAOodA2MnolPxLdCbSazJXy6w0/edit) I am unable to proceed. It wants me to create a new org entirely, is this normal? 

steps to reproduce: 
- logged in with admin gmail account. 
- can see my collectives
  - <img width="333" alt="Screenshot 2023-05-19 at 7 46 15 PM" src="https://github.com/omigroup/omigroup/assets/63426722/966b8cb9-6ee1-4e48-8070-60e0f302ff5e">
- clicking on [this link](https://opencollective.com/foundation/apply) (per instruction) takes me to a signup form 
- the form only allows me to create a new org 
  - <img width="1478" alt="Screenshot 2023-05-19 at 7 47 42 PM" src="https://github.com/omigroup/omigroup/assets/63426722/b2f0ddeb-558f-4757-a19f-ec18c91374ea">
- blocker: is this what we want? Do I create something new using [this information](https://docs.google.com/document/d/1_i1abJ0Log9UyqEjQiAOodA2MnolPxLdCbSazJXy6w0/edit) (thanks for providing this information btw) or can I convert our existing account? 👀 apologies if this was discussed.

**Comment by mrmetaverse** [comment]
![image](https://github.com/omigroup/omigroup/assets/63426722/a46f65ce-78b8-4fd7-8482-370635f6a86e)

**Comment by mrmetaverse** [comment]
![image](https://github.com/omigroup/omigroup/assets/63426722/dc032475-b2b3-47e3-8939-115d0a7913d9)

**Comment by mrmetaverse** [comment]
Once approved, we will request the migration of our funds from the existing account to the new one. 

The URL slug will change from /omigroup to /openmetaverse, unless I can get them to merge URL slugs too.

**Comment by mrmetaverse** [comment]
moving back to blocked. this is getting really confusing and I am not getting the support I need. Furthermore, I am not comfortable using OC at all if we cannot get better about submitting receipts on time. I cannot be expected to do that and all the other things I do. I need help. 

<img width="468" alt="Screenshot 2023-07-07 at 9 22 32 AM" src="https://github.com/omigroup/omigroup/assets/63426722/81ffbea7-3405-4769-a2ac-89ca39848f8e">

<img width="501" alt="Screenshot 2023-07-07 at 9 22 46 AM" src="https://github.com/omigroup/omigroup/assets/63426722/a7152f9a-7f8c-43fc-94bc-6c84f3fbbb01">

<img width="514" alt="Screenshot 2023-07-07 at 9 23 02 AM" src="https://github.com/omigroup/omigroup/assets/63426722/2f1ed3dc-4628-4d43-8ed5-c5a334be7559">

<img width="590" alt="Screenshot 2023-07-07 at 9 23 22 AM" src="https://github.com/omigroup/omigroup/assets/63426722/70b0f296-3e53-4e45-b156-2845959f401c">

<img width="543" alt="Screenshot 2023-07-07 at 9 23 40 AM" src="https://github.com/omigroup/omigroup/assets/63426722/705d8b56-60c9-4c86-be7f-f4de5bf8f837">

The [Process for switching fiscal hosts](https://docs.opencollective.com/help/collectives/change-fiscal-host#what-is-the-process-for-changing-fiscal-hosts) contradicts our previous instructions. We were told to create a second account and that the funds would be transferred and our old OMI Open Collective would be closed. Now they want us to "zero the account first". 

<img width="559" alt="Screenshot 2023-07-07 at 9 25 13 AM" src="https://github.com/omigroup/omigroup/assets/63426722/fe881d0b-d6de-4897-bcf2-9bf182e59fc1">


At this point, I am feeling a bit overwhelmed with the process. None of the steps have been as easy as predicted during planning. It is not straightforward, and I can only check email when I have time away from work. I am moving this to blocked unless someone else wants to help unblock this.

**Comment by mrmetaverse** [comment]
We found a way forward today during our Champions Sync (thanks @fire and @musicscene )

**Comment by indiebio** [comment]
> We found a way forward today during our Champions Sync (thanks @fire and @musicscene )

Apologies if this is now old, I don't have knowledge of what the way forward was, looking at their [website](https://docs.opencollective.com/help/collectives/change-fiscal-host#what-is-the-process-for-changing-fiscal-hosts), it is possible to move the funds: "You have two options to zero your collective balance. You can either send the funds directly to the new host or manually zero your collective balance. "
That page then underlines the steps to take for the first option.

My question without reading further is does the 501c3 have to be approved before you can transfer your funds to this new host?

**Comment by indiebio** [comment]
Out of curiosity, the link for the OCF [Eligibility requirements](https://docs.opencollective.foundation/getting-started/eligibility).

this created a brain trigger about "proper 501c3" ...

> not comfortable using OC at all
this comment now makes sense in the context of OC being a foundation for other sortof 501c3's 

> * cost: we give up a % of grants to our fiscal sponsor. Its a big job to hire an accountant for our own non profit. This cost is 10% per grant that comes in. It is a lot more complex to convert to proper 501c3 (10k to sign up, plus accountant fees, etc.)
I don't know what the amount of money is that is in OMI's coffers, but in my personal, non-US, experience I would recommend paying an hour of a lawyer/similar expert's time and getting their informed experienced opinion on this. In my personal, non-US, experience, doing the slightly more arduous admin heavy route to register the best entity, and then potentially paying a nominal sum to a bookkeeper to keep us in line has greater benefits in the long term. I am of course more than open to being corrected and I wasn't involved in the decision to go with OC so my opinion is not worth much :)

**Comment by mrmetaverse** [comment]
Moving back to todo. It is blocked but for new reasons. I sent an email last week to see what has changed (I think they are removing Cart services) and have not heard back.

**Comment by JoyceBettencourt** [comment]
@mrmetaverse in lieu of the recent announcement of the Open Collective Foundation (501c3) option closing down (https://blog.opencollective.com/open-collective-official-statement-ocf-dissolution/) OMI should probably check back and stop any transition process for now. 

The overall Open Collective platform isn't affected by any of this (Open Collective Foundation was a separate entity) - so if/until OMI has another plan to move forward with some other fiscal sponsorship option, we can keep finances in our original Open Collective account. 

Just make sure to reach out and check that they didn't start any transfer process since that could jeopardize OMI's access to funds.

OMI should also remove any mention of tax-deductible regarding donations. We have yet to be under the fiscal sponsorship of a 501c3 nonprofit, and being under the regular Open Collective org's 501c6 does not transfer any ability for us to say donations are tax deductible.

Maybe some rewording on OMI's site that: Donations to OMI help support our community and work we do in helping create the open metaverse. Donations are currently not tax-deductible.

**Comment by JoyceBettencourt** [comment]
We should still keep this issue viewable even if it gets moved back out of To Do. There are lots of good links and background for if/when we sort out what is next for OMI's entity/fiscal sponsor status.

**Comment by mrmetaverse** [comment]
The 501c3 version of OC has been terminated, so we are going to have to explore other options.

**Comment by mrmetaverse** [comment]
closed for https://github.com/omigroup/omigroup/issues/461
